package.json

```diff
@@ -18,16 +18,17 @@
   "author": "Relaying Services Team",
   "license": "MIT",
   "dependencies": {
-    "@rsksmart/rif-relay-contracts": "0.0.2-alpha.10",
-    "@rsksmart/rif-relay-common": "0.0.2-alpha.10",
     "@rsksmart/rif-relay-client": "0.0.2-alpha.8",
+    "@rsksmart/rif-relay-common": "0.0.2-alpha.10",
+    "@rsksmart/rif-relay-contracts": "0.0.2-alpha.10",
     "@truffle/abi-utils": "~0.2.3",
     "@truffle/hdwallet-provider": "1.6.0",
     "@trufflesuite/web3-provider-engine": "15.0.13-1",
-    "patch-package": "~6.4.6",
+    "@types/chai-http": "^4.2.0",
```

Comment:
Why do we need a type dependency among the `dependencies` instead of the `devDependencies`?